Remotion/Web/Core/WebStringConverter.cs

```diff
@@ -0,0 +1,148 @@
+
+    private static readonly Lazy<MethodInfo> s_createFromTextMethodInfo = new(
+        () =>
+        {
+          var methodInfo = typeof (WebString).GetMethod (nameof (WebString.CreateFromText), BindingFlags.Static | BindingFlags.Public)!;
+          Assertion.DebugIsNotNull (
+              methodInfo.GetParameters().SingleOrDefault (p => p.ParameterType == typeof (string)),
+              "The parameters of '{0}' have changed, please adapt '{1}' accordingly.",
+              nameof (WebString.CreateFromText),
+              nameof (WebStringConverter));
+          return methodInfo;
+        },
+        LazyThreadSafetyMode.ExecutionAndPublication);
+
+    private static readonly Lazy<MethodInfo> s_createFromHtmlMethodInfo = new(
+        () =>
+        {
+          var methodInfo = typeof (WebString).GetMethod (nameof (WebString.CreateFromHtml), BindingFlags.Static | BindingFlags.Public)!;
+          Assertion.DebugIsNotNull (
+              methodInfo.GetParameters().SingleOrDefault (p => p.ParameterType == typeof (string)),
+              "The parameters of '{0}' have changed, please adapt '{1}' accordingly.",
+              nameof (WebString.CreateFromHtml),
+              nameof (WebStringConverter));
+          return methodInfo;
+        },
+        LazyThreadSafetyMode.ExecutionAndPublication);
+
+    /// <inheritdoc />
+    public override bool CanConvertFrom (ITypeDescriptorContext context, Type sourceType)
+    {
+      return sourceType == typeof (string)
+             || base.CanConvertFrom (context, sourceType);
+    }
+
+    /// <inheritdoc />
+    public override bool CanConvertTo (ITypeDescriptorContext context, Type destinationType)
+    {
+      return destinationType == typeof (string)
+             || destinationType == typeof (InstanceDescriptor)
+             || base.CanConvertTo (context, destinationType);
+    }
+
+    /// <inheritdoc />
+    public override object? ConvertFrom (ITypeDescriptorContext context, CultureInfo culture, object? value)
+    {
+      if (value == null)
+        return null;
+
+      if (value is string @string)
+        return ConvertFromStringInner (@string);
+
+      return base.ConvertFrom (context, culture, value);
+    }
+
+    /// <inheritdoc />
+    public override object? ConvertTo (ITypeDescriptorContext context, CultureInfo culture, object? value, Type destinationType)
+    {
+      if (value == null)
+        return null;
+
+      var webString = (WebString) value;
+
+      if (destinationType == typeof (string))
+        return ConvertToStringInner (in webString);
+
+      if (destinationType == typeof (InstanceDescriptor))
+        return ConvertToInstanceDescriptor (in webString);
+
+      return base.ConvertTo (context, culture, value, destinationType);
+    }
+
+    private static object ConvertFromStringInner (string input)
+    {
+      if (input.StartsWith (c_textPrefix))
+        return WebString.CreateFromText (input.Substring (6));
```

Comment:
```suggestion
        return WebString.CreateFromText (input.Substring (c_textPrefix.Length));
```

De-duplication. Other occurrances, too, please